tests/pos/i18623a.scala

```diff
@@ -0,0 +1,20 @@
+final abstract class ForcedRecompilationToken[T]
+object ForcedRecompilationToken {
+  implicit def default: ForcedRecompilationToken["abc"] = null
+}
+
+object x {
+class GoodNoParens[T](implicit ev: ForcedRecompilationToken[T])
+}
+export x.GoodNoParens as BadNoParens
+
+// error
```

Comment:
Same here

Reply:
We usually leave the // error markers in the pos files. Less work to do, and it's useful to see what got fixed.